Dockerfile

```diff
@@ -1,8 +1,8 @@
 # syntax=docker/dockerfile:1
-FROM ocaml/opam:debian-12-ocaml-4.14@sha256:aeed2457dc910827bacd43f1f7bcc0759d6c6e6c676c062fd0c9adf7989965c2 AS build
+FROM ocaml/opam:debian-12-ocaml-4.14@sha256:6246731ea5d2cd3a57669027aae33184d30d424e41ff8219d05a214726ef7426 AS build
 RUN sudo ln -f /usr/bin/opam-2.2 /usr/bin/opam && opam init --reinit -ni
```

Comment:
```suggestion
RUN sudo ln -sf /usr/bin/opam-2.3 /usr/bin/opam && opam init --reinit -ni
```

Reply:
I was going to do this but I thought it better to change one thing at a time.